src/inverseRobot.js

```diff
@@ -7,7 +7,30 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotInverso = {};
+
+  for (const key in robot) {
+    const robotKey = robot[key];
```

Comment:
When you get key in object the result is value. So it's a little bit confusing that variable name is robotKey.